environment.yml

```diff
@@ -15,19 +16,21 @@ dependencies:
   - pip
   - plotly
   - pyarrow
-  - pygeos # used for pudl
+  - pygeos>=0.10,<0.14 # used for pudl
   - pytest
   - python>=3.10,<3.11 # require 3.10.x for compatibility with pudl
   - python-snappy # used for pudl
+  - qdldl-python==0.1.5,!=0.1.5.post2 # used for gridemissions, newer version not working as of 12/12/2022
```

Comment:
This is meant to patch https://github.com/singularity-energy/open-grid-emissions/issues/261